plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixConnectorFactory.java

```diff
@@ -56,6 +57,7 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
             Bootstrap app = new Bootstrap(
                     new JsonModule(),
                     new PhoenixClientModule(catalogName),
+                    new ColumnWithAliasFormatterModule(),
```

Comment:
nit: probably install this in `PhoenixClientModule` instead. The `ConnectorFactory` by convention provides things which the engine expects.

Reply:
maybe also change the Module name to match the way we change the class...

Reply:
yes, good catch. I missed that.